### PR TITLE
I18n package

### LINKED ISSUE #195: Replace i18n submodule with package
It might be easier for contributors to bump packages rather than submodules. Investigate publishing a package (via git?) and importing into the parent project

### PR DESCRIPTION
**Pull request by ann-kilzer**
## Resolves #195 

Uses the [package](https://github.com/ourjapanlife/findadoc-localization/packages/873560) instead of a submodule to incorporate latest i18n changes

### How to test

- Ensure the netlify build functions
- Pull the branch, run `yarn install`, then run the code
- You may need to `rm -rf i18n` from the root of your clone

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,18 +1,20 @@
 {
-  "name": "findadoc",
+  "name": "@ourjapanlife/findadoc-frontend",
```
Comment (ann-kilzer) [inline package.json:2]: This is github's standard for naming packages

File: nuxt.config.js [hunk 2]
```diff
@@ -134,7 +134,7 @@ export default {
           onlyOnRoot: true, // recommended for SEO
           useCookie: true,
         },
-        langDir: "i18n/locales",
+        langDir: "node_modules/@ourjapanlife/findadoc-localization/locales",
```
Comment (ann-kilzer) [inline nuxt.config.js:137]: 🤔 I probably should be importing this...

File: package.json [hunk 3]
```diff
@@ -1,18 +1,20 @@
...
   "scripts": {
     "dev": "nuxt dev",
     "build": "nuxt build",
     "start": "nuxt start",
     "generate": "nuxt generate",
-    "netlify": "git submodule update --init --recursive --remote && nuxt generate",
+    "netlify": "nuxt generate",
     "lint": "eslint . --ext .js,.vue,.json && prettier --write **/*.{js,vue,json,scss,md}",
-    "postinstall": "husky install"
+    "prepare": "husky install"
```
Comment (ann-kilzer) [inline package.json:13]: Apparently this is a much friendlier way to run the husky script, per their documentation. https://typicode.github.io/husky/#/?id=install

File: nuxt.config.js (hunk 2 above)
Comment (ann-kilzer) [reply nuxt.config.js:137]: Actually I have no idea how this will work 🤨

Comment (ermish) [reply nuxt.config.js:137]: interesting......I think this should work. depends on how it handles paths

File: README.md [hunk 4]
```diff
@@ -8,7 +8,7 @@
 
 [Join our slack!](https://join.slack.com/t/find-a-doc/shared_invite/zt-s4744a6o-MGaGHzLN5wB9aXeha3vdsQ)
 
-## Working with the submodule
+## Working with the submodule (will be deprecated after #195)
```
Comment (ermish) [inline README.md:11]: Should we just take this out alltogether?

File: package.json [hunk 5]
```diff
@@ -1,18 +1,20 @@
...
   },
   "dependencies": {
     "@nuxtjs/firebase": "^7.6.1",
+    "@ourjapanlife/findadoc-localization": "^1.0.0",
```
Comment (ermish) [inline package.json:17]: fun to see! 🙌

**Review by ermish — APPROVED** [review]
LGTM!

File: README.md (hunk 4 above)
Comment (ann-kilzer) [reply README.md:11]: Yes, but not until all the open PRs are merged. We need to get everyone synched on this

File: nuxt.config.js (hunk 2 above)
Comment (ann-kilzer) [reply nuxt.config.js:137]: It does work, It just feels dirty somehow

**Comment by ann-kilzer** [comment]
So Netlify is not finding the package:

```
11:19:15 PM: yarn install v1.22.4
11:19:16 PM: [1/4] Resolving packages...
11:19:16 PM: error An unexpected error occurred: "https://registry.yarnpkg.com/@ourjapanlife%2ffindadoc-localization: Not found".
```

I don't actually want to publish the package to yarn / npm, rather just publish it on GitHub. It's currently a public package.
 I have no idea if I can add an .npmrc to Netlify... 🤔

File: .npmrc [hunk 6]
```diff
@@ -0,0 +1 @@
+@ourjapanlife:registry=https://npm.pkg.github.com
```
Comment (ann-kilzer) [inline .npmrc:1]: It didn't work 😭

**Review by theyokohamalife — APPROVED** [review]
LGTM!

**Review by theyokohamalife — CHANGES_REQUESTED** [review]
Whoops. I should've looked closer before approving. Ran it locally and the package couldn't be found.

**Comment by ann-kilzer** [comment]
Yeah as you can see the Netlify build is failing. It actually does run locally for me, be sure to run `yarn install` whenever pulling new changes.

**Comment by ann-kilzer** [comment]
I'm at an impasse with Netlify and Yarn. Asked for help on Stack overflow: https://stackoverflow.com/questions/68226338/netlify-and-yarn-not-finding-github-package

If you can upvote my question, that would really help. Thanks!

File: package.json [hunk 7]
```diff
@@ -1,18 +1,21 @@
...
   "version": "1.0.0",
   "private": true,
+  "license": "BSD-3",
   "scripts": {
     "dev": "nuxt dev",
     "build": "nuxt build",
     "start": "nuxt start",
     "generate": "nuxt generate",
-    "netlify": "git submodule update --init --recursive --remote && nuxt generate",
+    "prenetlify": "echo //npm.pkg.github.com/:_authToken=$GITHUB_ACCESS_TOKEN >> ~/.yarnrc",
```
Comment (ann-kilzer) [inline package.json:11]: Apparently one has to edit the user level npmrc not the local one 

https://github.community/t/netlify-getting-401-from-github-package-registry-with-auth-token/16415/3

**Comment by ann-kilzer** [comment]
This is working end to end now, as you see the Netlify build passes. Please review again.

**Review by ermish — APPROVED** [review]
LGTM! Should simplify our startup effort! Thanks @ann-kilzer and everyone for your help!

**Comment by ermish** [comment]
We should update the readme in a separate PR

**Review by theyokohamalife — APPROVED** [review]
LGTM!

**Pull request merged**